setup.py

```diff
@@ -6,7 +6,7 @@
 
 
 REQUIREMENTS = [
-    'Django>=2.2',
+    'Django>=2.2,<4',
```

Comment:
```suggestion
    'Django>=2.2,<4.0',
```

Reply:
@marksweb I agree with @Aiky30 here. 

This is also high wagtail does it. 

https://github.com/wagtail/wagtail/blob/main/setup.py#L23

Reply:
Also `'Django>=2.2,<4.0',` looks nicer by reading in my opinion.

Reply:
Ok chaps, I've changed it 👍 

@Aiky30 @vinitkumar @crydotsnake